direct/core/src/test/java/cz/o2/proxima/util/TransformationRunner.java

```diff
@@ -60,6 +63,7 @@ public static void runTransformations(
     repo.getTransformations()
         .entrySet()
         .stream()
+        .filter(e -> !e.getValue().isTransactional())
```

Comment:
This looks a little bit weird and it's probably related to "transformation scope". Also javadocs says `Run all transformations in given repository.` which is not true for now.

Reply:
I think it is true, because all the transformations are 'running'. Transactional transformations run on client (and cannot be turned off) and server-side are running on server (actually one of them is the transformation that makes the client-side transformations work). So from the outside, this call really makes *all* transactions running.